ruby/ql/lib/codeql/codeql-pack.release.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Why are change notes and the codeql-pack.release.yml file stored in an arbitrary sub folder (`codeql`) containing also QL code instead of the pack root folder? In contrast to other languages that use `semmle.code`,  Ruby uses `codeql`  as top-level package name. It feels wrong to "pollute" the QL package folders with qlpack metadata.

Reply:
Oh, thanks for pointing that out! We'll either change the name of the directory where the extra pack metadata is collected, or just store it in the root (which would mean that there would be a top-level `change-notes` directory in your pack, but that actually seems more discoverable than `codeql/change-notes`).

Reply:
Fixed.